Menu/MenuBuilder.php

```diff
@@ -153,6 +102,8 @@ public function createSidebarMenu()
      * Sets the request the service.
      *
      * @param Request $request
+     *
+     * @deprecated since 2.4. To be removed in 3.0
```

Comment:
Please remove this method. This method was introduced only in dev-master. See #3065.